src/Makefile.am

```diff
@@ -150,12 +155,24 @@ apps_plugin_SOURCES = \
 
 if FREEBSD
```

Comment:
hmm, just seen this one, I believe better to include all sources and conditional within based on config.h settings.